Tools/environment_install/install-ROS-ubuntu.sh

```diff
@@ -211,6 +211,23 @@ sudo rosdep init || true
 rosdep update
 # Use this to install dependencies of packages in a workspace
 # rosdep install --from-paths src --ignore-src --rosdistro=${ROS_DISTRO} -y
+
+git clone https://github.com/SwiftGust/ardupilot_gazebo
+cd ardupilot_gazebo
+mkdir build
+cd build
+cmake ..
+make -j4
```

Comment:
Why limit to 4 or include `-j`? I have 8 cores and 16 virtual. Others will have different #s of cores. 

```suggestion
make -j4
```

Reply:
Thanks for the review @hendjoshsr71!
should I change it to - `make -j`? It will then not limit the processes

Reply:
I've remove the "-j4" in the alternative PR